.github/workflows/packages_tests.yaml

```diff
@@ -56,5 +56,7 @@ jobs:
                 if: ${{ github.event_name == 'pull_request' }}
 
             -   run: composer install --ansi
+            # temporary to give me access to merge
+            -   run: composer require doctrine/inflector:^2.0.6
```

Comment:
@TomasVotruba this is temporary to give me merge access as it used by rector-symfony

Reply:
cleaned tweak on https://github.com/rectorphp/rector-src/pull/5202